.devcontainer/scripts/run-devcontainer.sh

```diff
@@ -19,18 +19,29 @@ cd "$repo_root"
 
 container_name="$repo_name.$(stat -c%i "$repo_root/")"
 
+# Be sure to use the host workspace folder if available.
+workspace_root=${LOCAL_WORKSPACE_FOLDER:-$repo_root}
+
+if [ -e /var/run/docker-host.sock ]; then
```

Comment:
This makes it so we can use this script for local testing even inside the devcontainer.